lib/src/main/java/org/team100/lib/localization/PoseEstimationHelper.java

```diff
@@ -27,17 +27,20 @@ public class PoseEstimationHelper {
     private static final Telemetry t = Telemetry.get();
     private static final String m_name = Names.name(PoseEstimationHelper.class);
 
-    public static Translation2d cameraRotationToRobotRelative(Transform3d cameraInRobotCoordinates, Rotation3d cameraObject) {
+    public static Translation2d cameraRotationToRobotRelative(Transform3d cameraInRobotCoordinates, Rotation3d yawPitch) {
+            Translation2d roll = new Translation2d(yawPitch.getZ(),yawPitch.getY());
```

Comment:
so you have a **TRANSLATION** called **ROLL**.   which is made from a variable that has the worlds **YAW** and **PITCH**.  what does this mean?  what does "YAWPITCH" mean?  what coordinate system is in use here?